README.md

```diff
@@ -110,3 +110,39 @@ through Django shell (`./compose run --rm api manage shell`), use the following
 >>> user.groups.add(pe_group)
 >>> user.save()
 ```
+
+## Pulp 3 Installer
+
+#### 1. Clone the GitHub repos
+
+```console
+$ git clone https://github.com/ansible/galaxy_ng.git
+$ git clone https://github.com/pulp/pulplift.git
+```
+
+#### 2. Setup pulplift
+
+```console
+$ cd pulplift
+$ git submodule update --init
+$ cp example.dev-config.yml local.dev-config.yml
+```
+
+Uncomment `pulp-ansible`, `galaxy-pulp` and `galaxy-ng` on local.dev-config.yml:
```

Comment:
`local.dev-config.yml` is for the `source` boxes specifically.

Reply:
it needs to be source, because `galaxy-pulp` bindings will only exist locally

Reply:
Also, vagrant-sshfs becomes necessary when using source boxes.

See my updated pulplift instructions. https://github.com/pulp/pulplift/pull/77/files